src/main/resources/org/jenkinsci/plugins/workflow/steps/durable_task/ShellStep/help-script.html

```diff
@@ -9,4 +9,42 @@
     Otherwise the system default shell will be run, using the <code>-xe</code> flags
     (you can specify <code>set +e</code> and/or <code>set +x</code> to disable those).
     </p>
-</div>
\ No newline at end of file
+    <p>A few examples of usage include:</p>
+
+    <b>Creating an output folder</b>
+    <p><code>sh "mkdir -p output"</code></p>
+
+    <b>Returning the current directory Pipeline is running in.</b>
+    <p><code>sh "ls -la ${pwd()}"</code></p>
```

Comment:
Simplify the example to use the bash built-in `pwd`.
```suggestion
    <b>Reporting the current directory of the Pipeline step.</b>
    <p><code>sh 'pwd'</code></p>
```